k8s/rabbitmq/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,60 @@
 actions:
-- name: dummy test
+- name: RabbitMQ test
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-rabbitmq-svc.${NAMESPACE}.svc.cluster.local"
+      PORT=15672
+      USERNAME=''
+      PASSWORD=''
+      FAILED=0
+      DEBUG=1
+
+      # getting RabbitMQ username
+      echo "Getting RabbitMQ username."
+      USERNAME=$(kubectl get sts -oyaml \
```

Comment:
Generally, we should use benefits that Kubernetes give us. Please move this to the `tester.yaml.template` file.

Example:

https://github.com/GoogleCloudPlatform/click-to-deploy/blob/d242d7eaa4fcce804f6a9d03cf569a901a5cf9be/k8s/wordpress/apptest/deployer/manifest/tester.yaml.template#L36-L41

Reply:
I agree that using the environment variables declared in Kubernetes manifest would be a more readable solution here, especially in case of reading the username. Could you please use that pattern?